cmake/compiler/clang/target.cmake

```diff
@@ -1,6 +1,7 @@
 # Configuration for host installed llvm
 #
 
+# TODO: Address PR #13580
```

Comment:
Can you add the keyword "duplication"?

Reply:
I think duplication of configuration like this line is acceptable and tends to be expected. The real duplication issue is IMHO further down with the `foreach` **code** / logic. See comment in previous PR #13538